Source/Charts/Utils/ChartUtils.swift

```diff
@@ -226,7 +226,7 @@ extension CGContext {
         NSUIGraphicsPopContext()
     }
 
-    internal func drawMultilineText(_ text: String, at point: CGPoint, constrainedTo size: CGSize, anchor: CGPoint, knownTextSize: CGSize, angleRadians: CGFloat, attributes: [NSAttributedStringKey : Any]?)
+    func drawMultilineText(_ text: String, at point: CGPoint, constrainedTo size: CGSize, anchor: CGPoint, knownTextSize: CGSize, angleRadians: CGFloat, attributes: [NSAttributedStringKey : Any]?)
```

Comment:
how about we make these two `drawMultilineText` as open? I don't see why only internal.

Reply:
First, it's always been internal. Second, we expose a way to extend the draw methods by wrapping them in another open method. I think it's safe to keep it internal. If we get requests to expose it, maybe we can change our mind then.

Reply:
Also, I don't think it makes sense to have any of these be open, because they would have to subclass `CGContext` which isn't recommended.